uniffi_bindgen/src/bindings/kotlin/templates/RecordTemplate.kt

```diff
@@ -1,5 +1,6 @@
 data class {{ rec.name()|class_name_kt }} (
     {%- for field in rec.fields() %}
+    {% if rec.contains_unsigned_type(ci) %}@ExperimentalUnsignedTypes{% endif %}
```

Comment:
ah good catch. I actually had a separate method `rec.contains_unsigned_type(field)` that would do it correctly based on the type but was trying to consolidate the methods as much as I  can and missed that this would happen. I might quickly see how adding it to the whole data class would be vs individual fields though it'd be a little less convenient for the consumer